lib/reverse_sentence.rb

```diff
@@ -1,6 +1,23 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n) because the amount of time it will take will increase as the length of the input increases.
+# Space complexity: O(n) because the space increases as the inout increases.
```

Comment:
You were asked to try to do this in place which mean O(1) space complexity.  Can you see a way?